src/api/procedures/modifyMultiSig.ts

```diff
@@ -0,0 +1,154 @@
+import BigNumber from 'bignumber.js';
+
+import { PolymeshError, Procedure } from '~/internal';
+import { ErrorCode, MultiSig, Signer, TxTags } from '~/types';
+import { ProcedureAuthorization } from '~/types/internal';
+import { signerToSignatory, signerToString, stringToAccountId } from '~/utils/conversion';
+import { checkTxType } from '~/utils/internal';
+
+/**
+ * @hidden
```

Comment:
This should be `procedure/types` and not be hidden. If you want only `multiSig` in this can be set to hidden